### ISSUE TITLE
npm WARN enoent ENOENT: no such file or directory,open 'C:\Users\Adam.Arbico-PC\git\parse-server-example\node_modules\agent-base\package.json'

### ISSUE THREAD

**Issue by giri1991** [npm/npm#17687]
#### I'm opening this issue because:

  - [ ] npm is crashing.
  - [ 1] npm is producing an incorrect install.
  - [ ] npm is doing something I don't understand.
  - [ ] Other (_see below for feature requests_):

#### What's going wrong?

Hello, whilst installing npm within a parse server example I am getting the following errors:

> $ npm install
> npm WARN enoent ENOENT: no such file or directory, open 'C:\Users\Adam.Arbico-PC\git\parse-server-example\node_modules\agent-base\package.json'
> npm WARN enoent ENOENT: no such file or directory, open 'C:\Users\Adam.Arbico-PC\git\parse-server-example\node_modules\aws-sdk\package.json'
> npm WARN enoent ENOENT: no such file or directory, open 'C:\Users\Adam.Arbico-PC\git\parse-server-example\node_modules\apn\package.json'
> npm WARN enoent ENOENT: no such file or directory, open 'C:\Users\Adam.Arbico-PC\git\parse-server-example\node_modules\bcrypt\package.json'
> npm WARN enoent ENOENT: no such file or directory, open 'C:\Users\Adam.Arbico-PC\git\parse-server-example\node_modules\body-parser\package.json'
> npm WARN enoent ENOENT: no such file or directory, open 'C:\Users\Adam.Arbico-PC\git\parse-server-example\node_modules\busboy\package.json'

I have used "npm init" and created a package.json. However I have checked each of these file paths and there are no package.jsons present in most of them. How do I created a package.json for each of them, they have package.jsons in their sub folders and they all look different, I don't know where to even begin with this.

Thanks

**Comment by Butterfly** [comment]
Hi, @giri1991 Did you do anything fancy in your initial `npm` setup? I found your issue because I was hoping I could help someone with something I just learned from my own mistake. My apologies if it doesn't help, but what happened for me was that I received this message:

> npm WARN saveError ENOENT: no such file or directory, open '/home/elf/.git-me/[package-name]/package.json'
npm WARN enoent ENOENT: no such file or directory, open '/home/elf/.git-me/[package-name]/package.json'

I'd go in and look and see that, no, `package.json` was not there just like yours and just as that warning says.

I had seen a tip recently about `npm init`. A few seconds into attempting that route, I KNEW that's not what everyone else was doing because it was taking too much time (and *brain activity*, grin).

That's when I remembered that I [set mine up to install "globally"](https://docs.npmjs.com/getting-started/fixing-npm-permissions). That's where mine was found. It's filed kind of deep under a directory I named "npm-g" while following along with those global permissions instructions. If I open my terminal in that deep directory, my installation works just right now.

Again, my apologies if this doesn't help. Maybe your issue will still eventually help someone else now. :smile:
